### PR TITLE
New package: ptunnel-ng-1.42

### PR DESCRIPTION
**Pull request by mamins1376**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures:
  - armv6l-musl (cross)

### CODE REVIEW

File: srcpkgs/ptunnel-ng/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'ptunnel-ng'
+pkgname=ptunnel-ng
+version=1.42
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --libdir=/usr/lib --host=${XBPS_CROSS_TRIPLET}"
+short_desc="TCP forwarder and proxy used for ICMP/UDP tunneling"
+maintainer="Amin Sameti <mamins1376@gmail.com>"
+license=BSD
```
Comment (classabbyamp) [inline srcpkgs/ptunnel-ng/template:9]:
```suggestion
license=BSD-3-Clause
```

File: srcpkgs/ptunnel-ng/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'ptunnel-ng'
+pkgname=ptunnel-ng
+version=1.42
+revision=1
+build_style=configure
```
Comment (classabbyamp) [inline srcpkgs/ptunnel-ng/template:5]:
```suggestion
build_style=gnu-configure
```

File: srcpkgs/ptunnel-ng/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+configure_args="--prefix=/usr --libdir=/usr/lib --host=${XBPS_CROSS_TRIPLET}"
+short_desc="TCP forwarder and proxy used for ICMP/UDP tunneling"
+maintainer="Amin Sameti <mamins1376@gmail.com>"
+license=BSD
+homepage=https://github.com/utoni/ptunnel-ng
+distfiles=${homepage}/archive/refs/tags/v${version}.tar.gz
```
Comment (paper42) [inline srcpkgs/ptunnel-ng/template:11]:
```suggestion
homepage="https://github.com/utoni/ptunnel-ng"
distfiles="https://github.com/utoni/ptunnel-ng/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/ptunnel-ng/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/ptunnel-ng/template:9]: + add quotes

File: srcpkgs/ptunnel-ng/template (hunk 2 above)
Comment (paper42) [reply srcpkgs/ptunnel-ng/template:5]: and then you might be able to remove $configure_args

**Comment by mamins1376** [comment]
Rebased after applying fixes. All set?

**Comment by mamins1376** [comment]
I sent a review request. Is there anything else I should do?

**Pull request closed without merging**